scripts/world_bank/health_nutrition_population/population/README.md

```diff
@@ -0,0 +1,102 @@
+Importing World Bank Population Data into Data Commons
+Author: aarushitiwari@
+
+About the Dataset
+The world bank HNP dataset contains information about population, nutrition and health statistics of all countries, updated annually from all countries and marked special regions 
+The dataset from 1960 onwards is available online. For this import, the series selected are all population statistics – male and female, age 00 to 25.
+
+
+Size: 5447kb
+
+Format: Json (access is directly from browser)
+
+Time period: Annual, from 1960
+
+Granularity: Country level
+
+Variables: 
+
+Age: 26
+      
+Gender:2
+
+
+Population
+265 per age per gender
```

Comment:
265 countries?

Reply:
265 including groups of countries, such as Africa eastern and southern, etc. were there in the original dataset